src/components/SocialLinks-Color/index.js

```diff
@@ -84,9 +86,9 @@ const SocialLinksColor = () => {
             target="_blank"
             rel="noreferrer"
           >
-            <img
+            <StaticImage
               className="docker"
-              height="30px"
+              height={30}
```

Comment:
If we have the classes for each icon, then why define the height as an attribute as well?
Can't we add the same in the stylesheets directly?